ts/components/conversation/message/message-content/MessageAttachment.tsx

```diff
@@ -186,6 +186,7 @@ export const MessageAttachment = (props: Props) => {
       highlight={highlight}
       selected={selected}
       className={'module-message__generic-attachment'}
+      onClick={onClickOnGenericAttachment}
```

Comment:
@KeeJef You probably need to add a check for if `pending` is true (see line 191) then just return and don't do the onClick action. You should edit the onClickOnGenericAttachment directly.

Otherwise people will click on attachments while they are still loading or being processed.

Reply:
Resolved